docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -50,22 +42,16 @@ CONTROLLER3_SSH=""
 ----
 
 .Procedure
-//kgilliga: How is this different from the step in the Prerequisites section? In other words, why is one a prerequisite while the following 4 steps are in the main procedure?
-You can stop {OpenStackShort} services at any moment, but you might leave your environment in an undesired state. You should confirm that there are no ongoing operations.
 
-1. Connect to all the controller nodes.
-2. Remove any constraints between infrastructure and {OpenStackShort} control plane
-services.
-3. Stop the control plane services.
-4. Verify the control plane services are stopped.
+. Connect to all the Controller nodes.
 
```

Comment:
It's unclear to me whether steps 1-4 are steps, notes, or examples.

- In step 1 and 2, we don't tell customers how to do these actions.
- Step 3 seems more like a note than a step? And how do you check how the `cinder-backup` service is running and stop it?
- Step 4 seems like a conditional step that applies to customers who enable CephFS through NFS. Is that correct?